nautobot/extras/forms/forms.py

```diff
@@ -2070,6 +2071,23 @@ class Meta:
 #
 # Webhooks
 #
+class WebhookBulkEditForm(NautobotBulkEditForm):
+    """Bulk edit form for Webhook objects."""
+
+    pk = forms.ModelMultipleChoiceField(queryset=Webhook.objects.all(), widget=forms.MultipleHiddenInput())
+    enabled = forms.BooleanField(required=False)
+    type_create = forms.BooleanField(required=False)
```

Comment:
These probably all need to use `forms.NullBooleanField(required=False, widget=BulkEditNullBooleanSelect)` so that the bulk-edit distinguishes between "set to False" and "do not change" actions.